packages/plugins/content/slate/src/plugins/lists.js

```diff
@@ -24,7 +24,6 @@
 import React from 'react'
 import ListIcon from '@material-ui/icons/List'
 import OrderedListIcon from '@material-ui/icons/FormatListNumbered'
-import createListPlugin from 'slate-edit-list'
```

Comment:
Is `slate-edit-list` not compatible with the latest slate or what is the reason for removing it?